test/integration/connectors/test_delta_table_s3.py

```diff
@@ -0,0 +1,91 @@
+import os
+from pathlib import Path
+
+import pytest
+from deltalake import DeltaTable
+from fsspec import get_filesystem_class
+
+from test.integration.connectors.utils.constants import (
+    DESTINATION_TAG,
+)
+from test.integration.utils import requires_env
+from unstructured_ingest.v2.interfaces import FileData, SourceIdentifiers
+from unstructured_ingest.v2.processes.connectors.delta_table import (
+    CONNECTOR_TYPE,
+    DeltaTableAccessConfig,
+    DeltaTableConnectionConfig,
+    DeltaTableUploader,
+    DeltaTableUploaderConfig,
+    DeltaTableUploadStager,
+    DeltaTableUploadStagerConfig,
+)
+
+
+def get_aws_credentials() -> dict:
+    access_key = os.getenv("S3_INGEST_TEST_ACCESS_KEY", None)
+    assert access_key
+    secret_key = os.getenv("S3_INGEST_TEST_SECRET_KEY", None)
+    assert secret_key
+    return {
+        "AWS_ACCESS_KEY_ID": access_key,
+        "AWS_SECRET_ACCESS_KEY": secret_key,
+        "AWS_REGION": "us-east-2",
+    }
+
+
+@pytest.mark.asyncio
+@pytest.mark.tags(CONNECTOR_TYPE, DESTINATION_TAG)
+@requires_env("S3_INGEST_TEST_ACCESS_KEY", "S3_INGEST_TEST_SECRET_KEY")
+async def test_delta_table_destination_s3(upload_file: Path, temp_dir: Path):
```

Comment:
Can we have an s3 and local test in here to replace the current `delta-table-s3.sh` and `delta-table.sh` test scripts? 
Once those are removed, will also have to update the `test-dest.sh` file to not include those.

Reply:
So I deleted 
- test_e2e/dest/delta-table-s3.sh
- test_e2e/dest/delta-table.sh
- test_e2e/python/test-ingest-delta-table-output.py

and updated test_e2e/test-dest.sh to not include those tests.